super/S_Category/bot.py

```
@@ -0,0 +1,374 @@
+    "100": "بوابة",
+    "101": "نقاش البوابة",
+    "828": "وحدة",
+    "829": "نقاش الوحدة",
+    "2600": "موضوع",
+    "1728": "فعالية",
+    "1729": "نقاش الفعالية",
+}
+
+User_tables = {}
+
+def add_Usertables(table, family):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)